src/reducers/starWarsReducer.js

```diff
@@ -1,13 +1,34 @@
-import /* we need our action types here*/ "../actions";
+import {FETCH_START, FETCH_SUCCESS, FETCH_FAILURE} from "../actions";
 const initialState = {
-  characters: []
+  characters: [],
+  fetching: false,
+  error: ''
```

Comment:
When there is no error, keep it set to `null` instead of an empty string `""` - `null` states more declaratively that there is no error, as opposed to having an 'error' that is composes of empty text.